modules/cluster-logging-uninstall.adoc

```diff
@@ -29,3 +29,48 @@ after the Operator instances are deleted:
 ----
 $ oc delete pvc --all -n openshift-logging
 ----
+
+. Optional: Remove the Cluster Logging Operator and Elasticsearch Operator:
+
+.. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+.. Click the Options menu {kebab} next to the Cluster Logging Operator select *Uninstall Operator*.
+
+.. Click the Options menu {kebab} next to the Elasticsearch Operator select *Uninstall Operator*.
+
+. Optional: Delete the Cluster Logging and Elasticsearch Custom Resource Definitions (CRD).
+
+.. Switch to the *Administration* -> *Custom Resource Definitions* page.
+
+.. On the *Custom Resource Definitions* page, click *ClusterLogging*.
+
+.. On the *Custom Resource Definition Details* page, select *Delete Custom Resource Definition* from the *Actions* menu.
+
+.. On the *Custom Resource Definitions* page, click *Elasticsearch*.
+
+.. On the *Custom Resource Definition Details* page, select *Delete Custom Resource Definition* from the *Actions* menu.
+
+. Optional: Use the CLI to remove the Cluster Logging and Elasticsearch operator groups.
++
+[source,terminal]
+----
+$ oc delete og cluster-logging -n openshift-logging
+----
++
+[source,terminal]
+----
+$ oc delete og openshift-operators-redhat -n openshift-operators-redhat
```

Comment:
We should be careful about this recommendation as its possible other RH global operators could be here